README.md

```diff
@@ -117,84 +146,97 @@ func getServicePrincipalToken() (adal.OAuthTokenProvider, error) {
+In the Go SDK, profiles are available under the `profiles/` path and their
+component API versions are aliases to the true service package under
+`services/`. You can use them as follows:
 
 ```go
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
-import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/resources/mgmt/..."
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/storage/mgmt/storage"
 ```
 
-We also provide two special profiles: `latest` and `preview`. These will always
-refer to the most recent stable or preview API versions for each service. For
-example, to automatically use the most recent Compute APIs, use one of the following
-imports:
+The 2017-03-09 profile is the only one currently available and is for use with Azure
+Stack. More profiles are under development.
+
+In addition to versioned profiles, we also provide two special profiles
+`latest` and `preview`. These *always* include the most recent respective stable or
+preview API versions for each service, even when updating them to do so causes
+breaking changes.
+
+The `latest` and `preview` profiles can help you stay up to date with API
+updates as you build applications. Since they are by definition not stable,
+however, they **should not** be used in production apps. Instead, choose the
+latest specific API version (or an older one if necessary) from the `services/`
+path.
+
+As an example, to automatically use the most recent Compute APIs, use one of
+the following imports:
 
 ```go
 import "github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
 import "github.com/Azure/azure-sdk-for-go/profiles/preview/compute/mgmt/compute"
 ```
 
-## Resources
+# Resources
 
 - SDK docs are at [godoc.org](https://godoc.org/github.com/Azure/azure-sdk-for-go/).
 - SDK samples are at [Azure-Samples/azure-sdk-for-go-samples](https://github.com/Azure-Samples/azure-sdk-for-go-samples).
-- SDK notifications are published via the [Azure update feed][].
+- SDK notifications are published via the [Azure update feed](https://azure.microsoft.com/updates/).
 - Azure API docs are at [docs.microsoft.com/rest/api](https://docs.microsoft.com/rest/api/).
 - General Azure docs are at [docs.microsoft.com/azure](https://docs.microsoft.com/azure).
 
+### Other Azure packages for Go
+
+- [Azure Storage Blobs](https://azure.microsoft.com/services/storage/blobs) - [github.com/Azure/azure-storage-blob-go](https://github.com/Azure/azure-storage-blob-go) 
+- [Azure Applications Insights](https://azure.microsoft.com/en-us/services/application-insights/) - [github.com/Microsoft/ApplicationInsights-Go](https://github.com/Microsoft/ApplicationInsights-Go)
+
 ## License
 
-Apache 2.0, see [LICENSE][].
+Apache 2.0, see [LICENSE](./LICENSE).
 
 ## Contribute
 
-See [CONTRIBUTING.md][]. 
+See [CONTRIBUTING.md](./CONTRIBUTING.md).
```

Comment:
Aren't we supposed to include a "Code of Conduct" sectio?

Reply:
maybe, it's in the Contributing guide I think. let's defer fixing this now though